Assets/Scripts/Models/Character.cs

```diff
@@ -357,6 +366,13 @@ public void WriteXml(XmlWriter writer)
         }
 
         writer.WriteAttributeString("needs", needString);
+
+        StringBuilder statsString = new StringBuilder();
+        foreach (KeyValuePair<string, Stat> stat in stats)
+        {
+            statsString.Append(stat.Value.statType).Append(";").Append(stat.Value.Value).Append(":");
+        }
+        writer.WriteAttributeString("stats", statsString.ToString());
```

Comment:
I know that it might be easier to make your own parser. But is better to follow the xml way and save the stats like:
```
<Stats>
  <Stat name="Strength" value="10" />
</Stats>
```
It will be easier to extend and read when we add more attributes to the stats like xp.